### ISSUE TITLE
Docs.rs doesn't seem to run anymore for my crate simconnect-sys

### ISSUE THREAD

**Issue by jcramb** [rust-lang/docs.rs#2523]
### Crate name

simconnect-sys

### Build failure link

https://docs.rs/crate/simconnect-sys/0.23.1/builds/1041337

### Additional details

I've had some trouble getting Docs.rs to build docs for my crate simconnect-sys. Initially it seemed to be failing due to not using the `vendored` feature as specified in the metadata, so I've now changed the crate to be vendored by default to see if I can get it building but even with that Docs.rs doesn't seem to be trying to build the docs for the new version 0.24.3. 

Can you please provide any guidance as to how I can get Docs.rs attempting to build for this crate again?

Thanks!

**Comment by syphar** [comment]
Sorry for the delayed response. 

This is probably another case of #797 or #1849 , where some build errors won't be surfaced to the user. 
This problem will be solved in #2467, which is nearly finished, I hope I can finish it up at rustfest this weekend. 

What can help debugging these issues is [`cargo docs-rs`](https://github.com/dtolnay/cargo-docs-rs), which tries to build the docs the same way we do. 

The error I'm getting is: 

```
Error: failed to parse `package.metadata.docs.rs` for simconnect-sys

Caused by:
    default-target: invalid type: sequence, expected a string
```

**Comment by syphar** [comment]
I'm closing this issue here because the error is in the metadata. 

I also just merged & deployed #2467, so future errors would be visible now
